### PR TITLE
USHIFT-3271: Prevent pod name-collisions in storage tests

### PR DESCRIPTION
**Pull request by copejon**

### CODE REVIEW

File: test/suites/storage/pvc-resize.robot [hunk 1]
```diff
@@ -28,9 +28,12 @@ Increase Running Pod PV Size
...
     [Documentation]    Prepare the cluster env and test pod workload.
+    ${ns}=    Create Unique Namespace
+    Set Test Variable    \${NAMESPACE}    ${ns}
     Oc Create    -f ${SOURCE_POD} -n ${NAMESPACE}
     Named Pod Should Be Ready    ${POD_NAME_STATIC}
 
 Test Case Teardown
     [Documentation]    Clean up test suite resources
     Oc Delete    -f ${SOURCE_POD} -n ${NAMESPACE}
+    Oc Delete    ns ${NAMESPACE}
```
Comment (pacevedom) [inline test/suites/storage/pvc-resize.robot:39]: Maybe use this one? https://github.com/openshift/microshift/blob/main/test/resources/kubeconfig.resource#L82

File: test/suites/storage/reboot.robot [hunk 2]
```diff
@@ -28,10 +28,12 @@ Rebooting Healthy System Should Keep Functional PVC
...
+    ${ns}=    Create Unique Namespace
+    Set Test Variable    \${NAMESPACE}    ${ns}
     Oc Create    -f ${SOURCE_POD} -n ${NAMESPACE}
     Named Pod Should Be Ready    ${POD_NAME_STATIC}
 
 Test Case Teardown
     [Documentation]    Clean up test suite resources
     Oc Delete    -f ${SOURCE_POD} -n ${NAMESPACE}
-    Named Pod Should Be Deleted    ${POD_NAME_STATIC}    timeout=120s
+    Oc Delete    ns ${NAMESPACE}
```
Comment (pacevedom) [inline test/suites/storage/reboot.robot:39]: Maybe use this one? https://github.com/openshift/microshift/blob/main/test/resources/kubeconfig.resource#L82

File: test/suites/storage/snapshot.robot [hunk 3]
```diff
@@ -78,6 +80,7 @@ Test Case Teardown
     Oc Delete    pvc snapshot-restore -n ${NAMESPACE}
     Named PVC Should Be Deleted    test-claim-thin
     Named PVC Should Be Deleted    snapshot-restore
+    Oc Delete    ns ${NAMESPACE}
```
Comment (pacevedom) [inline test/suites/storage/snapshot.robot:83]: Maybe use this one? https://github.com/openshift/microshift/blob/main/test/resources/kubeconfig.resource#L82

File: test/suites/storage/pvc-resize.robot (hunk 1 above)
Comment (copejon) [reply test/suites/storage/pvc-resize.robot:39]: Oh thanks! I didn't see that

**Review by pacevedom — COMMENTED** [review]
/lgtm

**Pull request merged**